entries/a/also.com.json

```
@@ -0,0 +1,17 @@
+{
+  "ALSO Schweiz AG": {
+    "domain": "also.com",
+    "additional-domains": [
+      "also.ch"
+    ],
+    "img": "also.com.png",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://www.also.com/ec/cms5/de_6110/6110/services/it-services/multifaktor-authentifizierung/index.jsp",
+    "recovery": "https://weblogin.also.com/auth/realms/also-customers/login-actions/reset-credentials?client_id=pegasos-shop-6110&tab_id=Ofq4_qMkLOU",
+    "keywords": [
+      "retail"
```

From viewing their website it doesn't look like ALSO is a retailer. Their own description of themselves as a  "_Information and Communications Technology Provider_" doesn't really fit any other our existing categories so `other` is probably the best fit (unless `iot` fits?):
```suggestion
      "other"
```

```suggestion
      "iot"
```